src/Driver/MySQL/Schema/MySQLColumn.php

```diff
@@ -150,6 +151,9 @@ class MySQLColumn extends AbstractColumn
         'json',
     ];
 
+    #[ColumnAttribute(['string', 'varbinary'])]
```

Comment:
Should `integer` types be here in MySQL case?

Reply:
@roxblnfk MySQL does not provide int size in field information. And we cannot create a column with the desired size. Our integer column sizes are specified fixed here:
https://github.com/cycle/database/blob/2.x/src/Driver/MySQL/Schema/MySQLColumn.php#L249

For example, this test will fail (if we allow changing the size):

```php
public function testIntegerWithSize(): void
{
    $schema = $this->schema('table');

    $column = $schema->integer('column', size: 5);
    $schema->save();
    $this->assertSameAsInDB($schema);

    $this->assertSame(5, $column->getSize());
}
```

<img width="476" alt="size" src="https://github.com/cycle/database/assets/67324318/c8482679-1750-4c3a-ad39-c95a2eff10cb">


And in MySQL, it's not the field size, but only the displayed width. More about this in the documentation:
https://dev.mysql.com/doc/refman/8.0/en/numeric-type-attributes.html

Do we need the ability to specify it? If so, for the correct operation of tracking column changes, we need to exclude size from the comparison.

BTW, the same situation applies to **datetime**.

Reply:
> need to exclude size from the comparison

If it isn't difficult lets do it

Reply:
Added, for integers and datetime, time, timestamp